eZ/Publish/Core/Repository/UserService.php

```diff
@@ -77,7 +77,8 @@ public function __construct(RepositoryInterface $repository, Handler $userHandle
             'defaultUserPlacement' => 12,
             'userClassID' => 4, // @todo Rename this settings to swap out "Class" for "Type"
             'userGroupClassID' => 3,
-            'hashType' => User::PASSWORD_HASH_MD5_USER,
+             // @todo Change back default if we want part of this in 6.x
```

Comment:
Actually no, we can do this as long as we make sure corresponding legacy work is doen as well: https://github.com/netgen/ezpublish-legacy/commit/d630485600f4fa7b3fded21316e3aae41636bfb9 (based on and is missing author info from: https://github.com/ezsystems/ezpublish-legacy/pull/1195)